### PR TITLE
[Ticktets] Monitoring Agent and SQL Queriesls

### PR DESCRIPTION
**Pull request by salaymeh**
implemented monitoring agent scripts and sql queries

### CODE REVIEW

File: linux_sql/.idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (emhuskic) [inline linux_sql/.idea/workspace.xml:1]: .idea folder should not be pushed to Github.

File: linux_sql/sql/ddl.sql [hunk 2]
```diff
@@ -1,8 +1,11 @@
+CREATE DATABASE host_agent;
+\c host_agent;
 CREATE TABLE IF NOT EXISTS PUBLIC.host_info (
```
Comment (emhuskic) [inline linux_sql/sql/ddl.sql:3]: There is no need to have PUBLIC.host_info. Only host_info name is enough.

File: linux_sql/sql/ddl.sql [hunk 3]
```diff
@@ -18,23 +21,4 @@ CREATE TABLE IF NOT EXISTS PUBLIC.host_usage (
...
-  "timestamp", host_id, memory_free,
-  cpu_idle, cpu_kernel, disk_io, disk_available
-)
-VALUES
-  (
-    '2019-05-29 16:53:28', 1, 256, 95,
-    0, 0, 31220
-  );
\ No newline at end of file
+);
```
Comment (emhuskic) [inline linux_sql/sql/ddl.sql:24]: Does the host_usage table have an id? Where is it? 
You can set the primary key constraint to be:
```
PRIMARY KEY (timestamp, host_id)
);
```

**Review by emhuskic — CHANGES_REQUESTED** [review]

File: linux_sql/.idea/workspace.xml (hunk 1 above)
Comment (salaymeh) [reply linux_sql/.idea/workspace.xml:1]: Will fix this! thank you

File: linux_sql/sql/ddl.sql (hunk 2 above)
Comment (salaymeh) [reply linux_sql/sql/ddl.sql:3]: Thank you, confused if I needed it or not so I kept it just incase. But will fix it.

File: linux_sql/sql/ddl.sql (hunk 3 above)
Comment (salaymeh) [reply linux_sql/sql/ddl.sql:24]: Will add a primary key for host_usage. I

**Review by salaymeh — COMMENTED** [review]
updated all the changes and added sql queries didn't realize it never saved to git last pull request.

**Pull request closed without merging**